db/migrate/20220610054628_create_post.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
Adding author_id in reference to the users table refers to the user_id. You can remove the user_id column because you're making use of the author_id

Reply:
This issue is well resolved! ✔️